src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -71,7 +63,7 @@ public String getColor() {
     }
 
     public List<Station> getStations() {
-        return Arrays.asList(upStation, downStation);
+        return sections.getStations();
```

Comment:
```suggestion
        return Collections.unmodifiableList(sections.getStations());
```

외부에서 수정못하도록 막아도 좋을것 같네요!

Reply:
리스트를 불변으로 내려주도록 개선하겠습니다! 추가로 Stations도 일급 컬랙션으로 내려주도록 수정하도록 하겠습니다!